src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -75,7 +75,7 @@ export class PropertyTemplateOutline extends Component {
 
   resourceTemplatePromises = (templateRefs) => {
     return Promise.all(templateRefs.map(rtId =>
-      getResourceTemplateFromServer('ld4p', rtId)
```

Comment:
Drop explicit `group` arg in order to isolate RT group knowledge in `sinopiaServer.js`.